apps/precursor-chemicals/sections/summary-data-sections.js

```diff
@@ -9,31 +9,27 @@ module.exports = {
         step: '/licence-holder-details',
         field: 'licence-holder-details',
         parse: (list, req) => {
-          const licenseHolderDetails = [];
-          licenseHolderDetails.push(req.sessionModel.get('company-name'));
-          if (req.sessionModel.get('company-number')) {
-            licenseHolderDetails.push(req.sessionModel.get('company-number').toUpperCase());
-          }
-          if (req.sessionModel.get('website-url')) {
-            licenseHolderDetails.push(req.sessionModel.get('website-url'));
-          }
-          licenseHolderDetails.push(req.sessionModel.get('telephone'));
-          licenseHolderDetails.push(req.sessionModel.get('email'));
-          return licenseHolderDetails.join('\n');
+          const licenseHolderDetails = [
+            req.sessionModel.get('company-name'),
+            req.sessionModel.get('company-number').toUpperCase(),
+            req.sessionModel.get('website-url'),
+            req.sessionModel.get('telephone'),
+            req.sessionModel.get('email')
+          ];
+          return licenseHolderDetails.filter(element => element).join('\n');
```

Comment:
I'm wondering if we will have a need to store `licenseHolderDetails` or an address as a gathered and filtered array in session for any reason. If so it might be better to filter on the array before the `return` statement.

Reply:
We've done this before and have used these compound stored values in email templates, for example, but because of the uncertainty about future use in this project, it makes sense to skip this step for now and revisit it later if necessary.

Reply:
Fine with me, thanks